scripts/linters/linter_utils.py

```diff
@@ -117,10 +117,6 @@ class BaseLinter(abc.ABC):
 
     @abc.abstractmethod
     def perform_all_lint_checks(self) -> List[concurrent_task_utils.TaskResult]:
```

Comment:
Just a general question, how did you identify cases where the type defs and docstrings didn't exactly match and how do we ensure that the types/docstrings are accurate going forward (e.g. is there a lint rule or similar)?

Reply:
I'm not solving that issue in this PR ... this case also falls into the abstract-base-class issue above. I had to remove the Returns docstring here since there's a pylint check that forbids it, though.